businessRules/users.js

```diff
@@ -2,14 +2,21 @@ const { v4 } = require('uuid');
 const md5 = require('md5');
 const { createUser } = require('../models/users');
 const { getUsers, setUsers } = require('../utils/storage');
+const {
+    checkIfUserNameProvided,
+    checkIfPasswordProvided,
+    checkIfUserNameAndPasswordMatches,
+    checkIfUserLoginSessionProvided,
+    checkIfUserLoginSessionMatches,
+    checkIfUserIdMatches,
+    checkIfValueProvided,
+    checkIfKeyProvided,
+} = require('../validation/userfunctionValidation');
 
 function registerUser(userName, password, firstName, lastName) {
-    if (!userName) {
-        throw new Error('Error. There was no username provided.');
-    }
-    if (!password) {
-        throw new Error('Error. There was no password provided.');
-    }
+    checkIfUserNameProvided(userName);
```

Comment:
A clearer function name would be: `isUserNameValid`

Reply:
Renamed. Commited.

Reply:
No new changes

Reply:
Should be now.